docs/ref/models/lookups.txt

```diff
@@ -180,9 +180,10 @@ following methods:
     The notation to use a lookup in an expression is
     ``<lhs>__<lookup_name>=<rhs>``.
 
-    This class doesn't follow the :ref:`Query Expression API <query-expression>`
-    since it has ``=<rhs>`` on its construction: lookups are always the end of
-    a lookup expression.
+    This class follows the :ref:`Query Expression API <query-expression>`
+    because it implements the ``as_sql`` and ``as_vendorname`` methods.
+    Different than other lookup expressions, it has ``=<rhs>`` on its
+    construction: lookups are always the end of a lookup expression.
```

Comment:
I think the whole `as_sql`/`as_vendorname` bit is too much. 

It seems to me, the intent would be captured by something like: 

```suggestion
    This class implements acts as query expression, but, since it has ``=<rhs>``
    on its construction, lookups must always be the end of a lookup expression.
```

(That we don't have `get_lookup` &co is kind of implied I think...)

Reply:
@caioariede  Can you take a look at Carlton's suggestion?

Reply:
@felixxm yes, will do!

Reply:
@carltongibson @felixxm the `implements acts` wording seems odd to me, but I'm not a native speaker. Does that sound right?

Reply:
I would chop `implements`.

Reply:
Agreed. I've updated it, thanks @felixxm